src/Avalonia.Visuals/Rendering/SceneGraph/Scene.cs

```diff
@@ -310,7 +310,7 @@ private bool FilterAndClip(IVisualNode node, ref Rect? clip)
                         clipped = !node.GeometryClip.FillContains(controlPoint.Value);
                     }
 
-                    if (!clipped && node.Visual is ICustomHitTest custom)
+                    if (!clipped && node.Visual is ICustomSimpleHitTest custom)
```

Comment:
`ICustomSimpleHitTest` only [applies to the immediate renderer](https://github.com/AvaloniaUI/Avalonia/blob/ef38e9dd3a25e513f449a51ef09508af65140841/src/Avalonia.Visuals/Rendering/ICustomSimpleHitTest.cs#L8-L10) and `SceneGraph` is a deferred renderer concept, so I don't think this change is correct. Why was it needed?

Reply:
Ah yeah not really necessary here, reverted